dist/modules/tab.js

```diff
@@ -48,6 +48,7 @@ angular.module('mgcrea.ngStrap.tab', []).provider('$tab', function() {
       if (self.$panes[newIndex].disabled) {
         navigatePane(newIndex, toLeft);
       } else {
+        $('[id="' + self.$panes[newIndex].$labeledBy + '"]').focus();
```

Comment:
This changes is not required because the code has already been there to set focus to element. You can see the “focusOn” directive at end of the file, there you can change the code for focus. There already we setting the focus to “li” element . So you can set focus to “a” element instead of "li" by changing it to element[0].children[0].focus()